app/src/main/java/com/zeapo/pwdstore/git/config/SshjSessionFactory.kt

```diff
@@ -10,6 +10,7 @@ import com.github.ajalt.timberkt.w
 import com.zeapo.pwdstore.utils.clear
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.runBlocking
+import net.schmizz.sshj.DefaultConfig
```

Comment:
This is unused I believe

Reply:
I actually forgot to reformat, which would have resolved this and other issues.